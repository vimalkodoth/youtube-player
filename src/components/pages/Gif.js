import React from "react";
import { css } from "@emotion/core";
import gif from './../../static/img/kid.gif';
import SubmitButton from '../SubmitButton';
import Image from '../Image';
import { useHistory } from "react-router-dom";

function Gif() {
    const history = useHistory();
    const onClickHandler = () => {
        history.goBack();
    }
    return (
        <div css={Wrapper}>
            <Image src={gif} />
            <SubmitButton onClick={onClickHandler}>
                Back
            </SubmitButton>
        </div>
    )
}

export default Gif;

const Wrapper = css`
    display: flex;
    flex-direction: column;
    height: 100vh;
    justify-content: center;
    align-items: center;
    border: 1px solid #fff;
`;