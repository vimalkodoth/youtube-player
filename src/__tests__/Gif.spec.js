import React from "react";
import { shallow } from "enzyme";
import Gif from './../components/pages/Gif';

const originalConsoleError = console.error;

describe("Form", () => {
    beforeEach(() => {
        console.error = jest.fn();
    });

    afterEach(() => {
        console.error = originalConsoleError;
    });

    it("renders correctly", () => {
        const component = shallow(
            <Gif to={() => jest.fn()} />
        );
        expect(component).toMatchSnapshot();
    });
});
