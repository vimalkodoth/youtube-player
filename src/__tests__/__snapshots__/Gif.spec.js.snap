// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form renders correctly 1`] = `
<EmotionCssPropInternal
  __EMOTION_LABEL_PLEASE_DO_NOT_USE__="Gif"
  __EMOTION_TYPE_PLEASE_DO_NOT_USE__="div"
  css={
    Object {
      "name": "1vl48lr-Wrapper",
      "styles": "display:flex;flex-direction:column;height:100vh;justify-content:center;align-items:center;border:1px solid #fff;;label:Wrapper;",
    }
  }
>
  <Image
    src={Object {}}
  />
  <SubmitButton
    onClick={[Function]}
  >
    Back
  </SubmitButton>
</EmotionCssPropInternal>
`;
